src/app/pages/talk/talk.component.ts

```diff
@@ -199,15 +199,35 @@ export class TalkComponent implements OnDestroy {
 
     this.slideIndex++;
     const page: string = this.structure.ORDER[this.slideIndex];
-    this.setPage(page, this.structure);
```

Comment:
Let's remove this code and revert it back to what it was.

I would prefer to filter the `{ visibility: false }` records when the JSON file is loaded. In the `/app/core/services/code.service.ts` there's a function `getStructure` that can be used to filter. I would pass in a boolean (default to false) that tells it whether to filter out the `{ visible: false }` objects. If we pass in `true` in the talk component, we'll never see those that aren't visible.

Remember, we need to update the Unit Tests, as well.

Reply:
Okay, I'll see what I can do with that route.

Reply:
I have made the necessary changes to getStructure so that the `{ visibility: false }` objects are never sent to the component.

> If we pass in true in the talk component, we'll never see those that aren't visible.

Is this not the required behaviour?